warehouse/config.py

```diff
@@ -270,6 +270,18 @@ def configure(settings=None):
         "PASSWORD_RESET_RATELIMIT_STRING",
         default="5 per day",
     )
+    maybe_set(
+        settings,
+        "warehouse.manage.oidc.user_provider_registration_ratelimit_string",
+        "USER_OIDC_PROVIDER_REGISTRATION_RATELIMIT_STRING",
+        default="10 per day",
+    )
+    maybe_set(
+        settings,
+        "warehouse.manage.oidc.ip_provider_registration_ratelimit_string",
+        "IP_OIDC_PROVIDER_REGISTRATION_RATELIMIT_STRING",
+        default="10 per day",
+    )
```

Comment:
This might be way too conservative, given CGNAT and other many-to-one-IP networking scenarios. Thoughts?

Reply:
This seems reasonable.